lib/lib-table-cluster/src/main/java/org/sagebionetworks/table/cluster/SqlQuery.java

```diff
@@ -210,8 +215,12 @@
 		}else{
 			this.includesRowIdAndVersion = false;
 		}
+
 		SQLTranslatorUtils.translateModel(transformedModel, parameters, columnTranslationReferenceLookup);
 		this.outputSQL = transformedModel.toSql();
+		if(outputSQL.contains("CURRENT_USER()")){  // Non-SQL function

```

Comment:
This substitution should go in the SqlTranslatorUtils.translateModel method and should not use the replace from the string but rather the utilities in the Model to build the value of the function.